UNITTESTS/CMakeLists.txt

```diff
@@ -114,6 +114,7 @@ set(unittest-includes-base
   "${PROJECT_SOURCE_DIR}/../features"
   "${PROJECT_SOURCE_DIR}/../platform/include"
   "${PROJECT_SOURCE_DIR}/../platform/include/platform"
+  "${PROJECT_SOURCE_DIR}/../platform/mbed-trace/include"
```

Comment:
Why the `mbed-trace` directory is renamed as `include`?

Reply:
We still have `mbed-trace/mbed_trace.h` inside `include/`. The change is to do with include path namespacing in the new build system being developed.